docs/src/@primer/gatsby-theme-doctocat/components/head.js

```diff
@@ -19,10 +20,12 @@ function Head(props) {
 
   let primerViewComponentsSrc
 
-  if(process.env.NODE_ENV === 'development') {
-    primerViewComponentsSrc = "http://localhost:4000/assets/primer_view_components.js"
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
What about something like

```javascript
const primerViewComponentsSrc = ( () => {
  if (process.env.NODE_ENV === 'development') {
    return 'http://localhost:4000/assets/primer_view_components.js'
  } else {
    return 'https://unpkg.com/@primer/view-components@latest/app/assets/javascripts/primer_view_components.js'
  }
})()
```

Reply:
Hmm I didn't add this or touch it so I'm gonna leave it for now